### PR TITLE
[Feature:Developer] Add Archived Sample Course

### PR DESCRIPTION
**Pull request by IDzyre**
### Please check if the PR fulfills these requirements:

* [ ] Documentation has been updated/added if relevant

### What is the current behavior?
Currently there are no courses, or way to make courses archived with the Setup Sample Courses scripts. #10778 required a sample archived course to verify during testing where the Self Registration courses show up on the My Courses page. 

### What is the new behavior?
A sample Archived course has been added, and the setup sample courses scripts have been updated to allow for archiving courses.

### CODE REVIEW

File: sbin/create_course.sh [hunk 1]
```diff
@@ -51,11 +51,23 @@ if [[ "$table_count" -ne "5" ]] ; then
     echo "ERROR: Submitty Master DB is invalid."
     exit
 fi
-
+archived=0
+while :; do
+    case $1 in
```
Comment (IDzyre) [inline sbin/create_course.sh:56]: This is kind of extraneous for just one case, but #10778 adds more.

**Comment by williamjallen** [comment]
I just kicked off [this](https://github.com/Submitty/Submitty/actions/runs/10948021715) vagrant up job to verify that the archived course can be built successfully.

**Comment by IDzyre** [comment]
> I just kicked off [this](https://github.com/Submitty/Submitty/actions/runs/10948021715) vagrant up job to verify that the archived course can be built successfully.

I will re-run a vagrant up job once #10964 is merged.

**Comment by williamschen23** [comment]
vagrant up is [here](https://github.com/Submitty/Submitty/actions/runs/10966734858)

**Review by williamjallen — APPROVED** [review]

**Pull request merged**